src/journal/test-compress.c

```diff
@@ -274,10 +275,12 @@ int main(int argc, char *argv[]) {
 
         char data[512] = "random\0";
 
-        char huge[4096*1024];
-        memset(huge, 'x', sizeof(huge));
+        _cleanup_free_ char *huge = NULL;
+
+        assert_se(huge = malloc(HUGE_SIZE));
+        memset(huge, 'x', HUGE_SIZE);
```

Comment:
Strictly speaking, this doesn't matter, but it's ugly: this should be `HUGE_SIZE-1 - strlen("HUGE=")` at offset `strlen("HUGE=")`, and swithched in order with the next line.

Reply:
I thought about this, but then the compiler or CPU may also be able to optimize away the redundant stores.